examples/integration/xcodeproj_targets.bzl

```diff
@@ -146,6 +146,14 @@ TVOS_BUNDLE_ID = IOS_BUNDLE_ID
 WATCHOS_BUNDLE_ID = "{}.watch".format(IOS_BUNDLE_ID)
 
 SCHEME_AUTOGENERATION_MODE = "all"
+SCHEME_AUTOGENERATION_CONFIG = xcschemes.autogeneration_config(
+    exclude_patterns = xcschemes.autogeneration_exclude_patterns(
+        target_name = [
```

Comment:
Should this be 
```suggestion
        target_names = [
```

Reply:
My intent was to interpret this as "list of regexes to match on `target_name` for each target", my concern with adding the `s` there is people thinking this is a list of targets instead of a list of regexes?

Reply:
What about `target_name_regex` or `target_name_patterns`. No strong feelings here though if you think as-is is fine

Reply:
Yeah, pondered about this when testing it. Maybe I'm trying to make it too customizable upfront and a single attribute is simpler than the nested one `"exclusion_patterns": { "target_name": [...] }`.

Also, technically the filtering happens in the scheme name so maybe the name should change too, i.e. simply:
```
scheme_autogeneration_config = {
    "scheme_name_exclusion_patterns": [ ... ]
}
```

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3027/commits/91a4d2884b8643dfb41400a901b7674ce67e2b48